src/main/java/io/github/haykam821/clutchpractice/clutch/ClutchType.java

```diff
@@ -16,7 +16,6 @@
 public abstract class ClutchType {
 	private final ItemStack icon;
 	private Text name;
-
```

Comment:
Make sure to keep spacing between methods and fields.

Reply:
New commit fixes this